doc/adr/0002-use-referer-header-for-id-acquisition.md

```diff
@@ -0,0 +1,32 @@
+# 2. Use referer header for ID acquisition
+
+Date: 2017-06-21
+
+## Status
+
+Accepted
+
+## Context
+
+The mechanism for directing people to the new GP Profile pages is to use an
+existing system where a banner is displayed on the current page. From this
```

Comment:
No, the banners are temporary. I feel there is enough information provided to understand the decision around using the header.